specification/cognitiveservices/data-plane/Language/preview/2022-03-01-preview/ConversationalAnalysis-authoring.json

```diff
@@ -0,0 +1,2898 @@
+          "description": "The prebuilts per language.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrebuiltsPerLanguageGetObject"
+          }
+        }
+      }
+    },
+    "PrebuiltsPerLanguageGetObject": {
+      "required": [
+        "language",
+        "prebuiltEntities"
+      ],
+      "type": "object",
+      "properties": {
+        "language": {
+          "description": "The language.",
+          "type": "string"
+        },
+        "prebuiltEntities": {
+          "description": "The prebuilts.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/PrebuiltGetObject"
+          }
+        }
+      }
+    },
+    "ProjectCreateObject": {
+      "required": [
+        "kind",
+        "settings",
+        "projectName",
+        "language"
+      ],
+      "type": "object",
+      "properties": {
+        "kind": {
+          "description": "Represents the project kind.",
+          "enum": [
+            "conversation",
+            "workflow"
+          ],
+          "type": "string"
+        },
+        "settings": {
+          "description": "Gets or sets the project settings.",
+          "$ref": "#/definitions/ProjectSettings"
+        },
+        "projectName": {
+          "description": "Gets or sets the new project name.",
+          "type": "string"
+        },
+        "multilingual": {
+          "description": "Gets or sets a value indicating whether gets or sets whether the project is multilingual or not.",
+          "type": "boolean"
+        },
+        "description": {
+          "description": "Gets or sets the new project description.",
+          "type": "string"
+        },
+        "language": {
+          "description": "Gets or sets the project language.",
+          "type": "string"
+        }
+      }
+    },
+    "ProjectDeletionJobStatusObject": {
+      "required": [
+        "jobId",
+        "createdDateTime",
+        "lastUpdatedDateTime",
+        "expirationDateTime",
+        "status"
+      ],
+      "type": "object",
+      "properties": {
+        "jobId": {
+          "description": "Gets or sets the job id.",
+          "type": "string"
+        },
+        "createdDateTime": {
+          "format": "date-time",
+          "description": "Gets or sets the created time.",
+          "type": "string"
+        },
+        "lastUpdatedDateTime": {
+          "format": "date-time",
+          "description": "Gets or sets the last updated time.",
+          "type": "string"
+        },
+        "expirationDateTime": {
+          "format": "date-time",
+          "description": "Gets or sets the expiration time.",
+          "type": "string"
+        },
+        "status": {
+          "description": "Gets or sets the job status.",
+          "enum": [
+            "unknown",
+            "notStarted",
+            "running",
+            "succeeded",
+            "failed",
+            "cancelled",
```

Comment:
>cancelled [](http://example.com/codeflow?start=13&length=9)

Canceled, see [https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#long-running-operations--jobs](https://github.com/microsoft/api-guidelines/blob/vNext/azure/Guidelines.md#long-running-operations--jobs) for example status names.